### PR TITLE
Remove partial doc {@template}s

### PR DESCRIPTION
**Pull request by srawlins**
I'm fairly certain these are broken; a `{@template}` directive must be followed by one or more lines and then `{@endtemplate}`.

Let me know if I'm mistaken 😁

### CODE REVIEW

**Comment by rrousselGit** [comment]
These should most likely be `@macro` instead and I forgot to rename the keyword while copy-pasting the template line

**Comment by rrousselGit** [comment]
The new lint is quite cool! Thanks for that :)

File: packages/riverpod/lib/src/framework/container.dart [hunk 1]
```diff
@@ -291,7 +291,6 @@ class ProviderContainer implements Node {
     );
   }
 
-  /// {@template riverpod.invalidate}
   void invalidate(ProviderOrFamily provider) {
```
Comment (rrousselGit) [inline packages/riverpod/lib/src/framework/container.dart:294]:
```suggestion
  /// {@macro riverpod.invalidate}
  void invalidate(ProviderOrFamily provider) {
```

File: packages/riverpod/lib/src/framework/container.dart [hunk 2]
```diff
@@ -310,7 +309,6 @@ class ProviderContainer implements Node {
     }
   }
 
-  /// {@template riverpod.refresh}
   State refresh<State>(Refreshable<State> provider) {
```
Comment (rrousselGit) [inline packages/riverpod/lib/src/framework/container.dart:312]:
```suggestion
  /// {@macro riverpod.refresh}
  State refresh<State>(Refreshable<State> provider) {
```

**Comment by srawlins** [comment]
Excellent, thanks much @rrousselGit !!

**Pull request merged**